src/test/mod.rs

```diff
@@ -560,7 +560,7 @@ fn get_config(config_file: Option<&Path>) -> Config {
         .read_to_string(&mut def_config)
         .expect("Couldn't read config");
 
-    Config::from_toml(&def_config, Path::new("tests/config/")).expect("invalid TOML")
```

Comment:
If I understand correctly, the folder `"tests/config/"` was ignored. Where is it ignored now ?

Reply:
```tests/config/``` includes no rs files now(only toml files).
So I think ```tests/config``` does not need to be ignored.
https://github.com/rust-lang/rustfmt/tree/master/tests/config

Reply:
indeed, thanks for checking!